bench/README.md

```diff
@@ -65,43 +68,123 @@ And open `http://localhost:9090/` to query and monitor the server-side metrics.
 
 ## Headless
```

Comment:
Steps for building docker image with DCL_NODE and DCL_VERSION parameters is missing

Reply:
Done! ✅